website/content/en/docs/best-practices/pod-security-standards.md

```diff
@@ -111,27 +111,98 @@ the namespace labeling as part of the manual install instructions. Following the
     pod-security.kubernetes.io/warn: privileged
 ```
 
+Following you will find a detailed description of how to configure and test your solutions. 
+The most straightforward way to ensure if your workloads can work on a restricted namespace is verifing if your solution can run in namespaces enforced as restricted. 
+
+**RECOMMENDED** ensure the desired behaviour via the e2e test. (_PS.:You will find code examples on the content shared_)
```

Comment:
Nit:
```suggestion
**NOTE**: It is recommended that you test the desired behavior as part of an e2e test suite. Examples of an e2e test for this will be shown in a later section.
```